### PR TITLE
Sergeyro/fix/remove builded icons

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4786437354

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
   chromatic-deployment:
     # Operating System
     runs-on: ubuntu-latest
-    # Job steps
     steps:
       - uses: actions/checkout@v1
       - name: Install dependencies
         run: npm ci
-        # 👇 Adds Chromatic as a step in the workflow
+      - name: Build react icons
```
Comment (shlomitc) [inline .github/workflows/chromatic.yml:17]: why is this needed? isn't it enough to to be ran as part of the regular build step in the package.json?

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: .github/workflows/chromatic.yml (hunk 1 above)
Comment (SergeyRoyt) [reply .github/workflows/chromatic.yml:17]: @shlomitc It isn't enough - it was crashing on all these steps before

Comment (shlomitc) [reply .github/workflows/chromatic.yml:17]: this is the only one I see that doesn't have any build actions at all, what command is it running behind the scenes that build the storybook?

File: .github/workflows/test-workflow.yml [hunk 2]
```diff
@@ -18,6 +18,8 @@ jobs:
           npm ci
         env:
           NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+      - name: Build react icons
```
Comment (shlomitc) [inline .github/workflows/test-workflow.yml:21]: in the same context as the other comment - then it sounds to me like it should be added to each of the commands and not only for the CI.

As a best practice - the CI steps should only be familiar with only one command from the package.json and not try to have some logic composition multiple commands

Comment (SergeyRoyt) [reply .github/workflows/test-workflow.yml]: Combined it via `ci` script in `package.json`

**Comment by SergeyRoyt** [comment]
Closed in favor of this solution
